templates/includes/breadcrumb_journey.html.twig

```diff
@@ -4,9 +4,9 @@
             <a class="govuk-breadcrumbs__link" href="{{ ccs_domain }}">Home</a>
         </li>
         <li class="govuk-breadcrumbs__list-item" id="govuk-breadcrumbs_search">
-            <a class="govuk-breadcrumbs__link" href="{{ ccs_domain }}/agreements/search?q={{ searchBy }}">Search
+            <a class="govuk-breadcrumbs__link" href="{{ ccs_domain }}/agreements/search?q={{ searchBy }}&feature=guidedmatch">Search
                 framework</a>
         </li>
         <li class="govuk-breadcrumbs__list-item" aria-current="page" id="govuk-breadcrumbs_guided_match">Guided match</li>
     </ol>
-</div>
\ No newline at end of file
+</div>ß
```

Comment:
Hi @adrianmos1980  - just checking - is this symbol meant to be here after the `</div>`?

Reply:
Hi @adrianmilne , thanks. I was a mistype , option+s instead of ctrl + s  , on a mac keyboard :)